stories/components/logger.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { StoryObj } from '@storybook/react';
+import { FifoLogger } from 'fifo-logger';
+
+import {
+  FifoLoggerDialog,
+  FifoLoggerProvider,
+} from '../../src/components/index.js';
+
+export default {
+  title: 'Components / Logger',
+  component: FifoLoggerDialog,
+};
+
+type Story = StoryObj<typeof FifoLoggerDialog>;
+
+const fifoLogger = new FifoLogger({ level: 'debug' });
+export const Control = {
+  argTypes: {
+    logs: {
+      table: {
+        disable: true,
+      },
+    },
+    onClose: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  args: {
+    isOpen: true,
+    logs: [
```

Comment:
Can you add a more representative set of logs, in particular with different levels